plugin/pkg/auth/authenticator/token/tokenfile/tokenfile_test.go

```diff
@@ -29,6 +29,8 @@ func TestTokenFile(t *testing.T) {
 	auth, err := newWithContents(t, `
 token1,user1,uid1
 token2,user2,uid2
+token3,user3,uid3,group1,group2
```

Comment:
if you used double quotes around a comma-separated list of groups, then this would not be a problem.